crates/extensions/c8y_mapper_ext/src/tests.rs

```diff
@@ -2786,6 +2786,7 @@ pub(crate) fn test_mapper_config(tmp_dir: &TempTedgeDir) -> C8yMapperConfig {
         config_snapshot: true,
         config_update: true,
         firmware_update: true,
+        device_profile: true,
```

Comment:
If this one was using `Capabilities::default()`, you wouldn't need to add a new line...


```suggestion
    let capabilities = Capabilities::default();
```